.github/workflows/desktop.release.reuse.yml

```diff
@@ -78,21 +78,25 @@ jobs:
 
       - run: yarn install --immutable
 
-      - name: Rebuild Electron Native Modules
-        run: yarn workspace @standardnotes/desktop rebuild:home-server
-
-      - run: yarn build:desktop
```

Comment:
This is the same as `yarn run webpack --config desktop.webpack.prod.js` which runs below I think?